lib/sdr_client/cli.rb

```diff
@@ -46,6 +49,11 @@ def self.start(command, options, arguments = [])
         puts SdrClient::Find.run(arguments.first, **options)
       when 'deposit', 'register'
         deposit(command, options, arguments)
+      when 'update'
+        raise "Expected a single druid argument, received #{arguments}" if arguments.size != 1 || !arguments.first.is_a?(String)
+
+        job_id = SdrClient::Update.run(arguments.first, **options)
+        poll_for_job_complete(job_id: job_id, url: options[:url]) # TODO: add an option that skips this
```

Comment:
We don't currently test this file and I'm not strongly inclined to start doing so in this PR, so we have a coverage ding right here.

Reply:
you could skip in in the simplecov configure options (see dor-services-app for example)

Reply:
I considered it! Here's why I didn't: it would send the signal that we are giving up on test coverage for that file. If we're hoping to have actual CLI users, I think we're going to want to add test coverage vs. a temporary bump in coverage for a green checkmark.

Reply:
And thank you for the review!